weed/filer/filechunk_section.go

```diff
@@ -61,6 +62,19 @@ func removeGarbageChunks(section *FileChunkSection, garbageFileIds map[string]st
 }
 
 func (section *FileChunkSection) setupForRead(group *ChunkGroup, fileSize int64) {
+	if section.isPrepared {
+		section.reader.fileSize = fileSize
+		return
+	}
```

Comment:
Why 65-68 is same with 73-76? Is it intentional or mistake?

Because 65-68 is not thread safe.